packages/grafana-ui/src/components/CustomScrollbar/CustomScrollbar.tsx

```diff
@@ -127,16 +140,68 @@ export class CustomScrollbar extends Component<Props> {
         // Before these where set to inherit but that caused problems with cut of legends in firefox
         autoHeightMax={autoHeightMax}
         autoHeightMin={autoHeightMin}
-        renderTrackHorizontal={this.renderTrackHorizontal}
-        renderTrackVertical={this.renderTrackVertical}
-        renderThumbHorizontal={this.renderThumbHorizontal}
-        renderThumbVertical={this.renderThumbVertical}
-        renderView={this.renderView}
+        renderTrackHorizontal={(passedProps: any) => this.renderTrackHorizontal(passedProps, styles)}
+        renderTrackVertical={(passedProps: any) => this.renderTrackVertical(passedProps, styles)}
+        renderThumbHorizontal={(passedProps: any) => this.renderThumbHorizontal(passedProps, styles)}
+        renderThumbVertical={(passedProps: any) => this.renderThumbHorizontal(passedProps, styles)}
+        renderView={(passedProps: any) => this.renderView(passedProps, styles)}
       >
         {children}
       </Scrollbars>
     );
   }
 }
 
-export default CustomScrollbar;
+export const CustomScrollbar = withTheme(UnthemedCustomScrollbar);
+CustomScrollbar.displayName = 'CustomScrollbar';
+
+const getStyles = stylesFactory(
+  (theme: GrafanaTheme, isPageScrollbar?: boolean): StylesInterface => {
+    return {
+      customScrollbar: css`
+        display: flex;
+        flex-grow: 1;
+        &:hover {
+          opacity: 0.8;
+          transition: opacity 0.3s ease-in-out;
+        }
```

Comment:
this makes the whole view get opacity & get darker

Reply:
okay. I guess we should remove this entirely then